src/module/actor/entity.js

```diff
@@ -293,17 +288,17 @@ export class OseActor extends Actor {
   }
 
   rollCheck(score, options = {}) {
```

Comment:
Unfortunately last I checked (via League Developers discord) there isn't support for click modifier keys (hold this button when there's a click) in Foundry's keybind API. It's possible there's a way to utilize the API to this end anyway, but because it's not the designed behavior I abandoned that idea